ani-cli

```diff
@@ -544,7 +564,15 @@ case $scrape in
 		search_results=$(search_anime "$query")
 		[ -z "$search_results" ] && die "No search results found"
 		anime_selection "$search_results"
-		episode_selection
+        if [ "$REPLY" != "^[0-9]+$" ] ; then
+            while [ "$REPLY" = "a" ]; do
+                search_another_anime
+            done
+            [ "$REPLY" = "q" ] && exit 0
+            [ "$REPLY" != "[aq]" ] && die "invalid choice"
+        else
+		    episode_selection
+        fi
```

Comment:
This could be 3 lines of `[ "$REPLY" = pattern ] && action`
Something like this could work
```suggestion
        [ "$REPLY" != "[0-9]+$" ] && episode_selection
        [ "$REPLY" = "a" ] && search_another_anime
        [ "$REPLY" = "q" ] && exit 0
        die "invalid choice"
```

Reply:
second time search is not working